Shared/Realm.Sync.Shared/Exceptions/SessionErrorException.cs

```diff
@@ -28,11 +28,13 @@ public class SessionErrorException : Exception
         /// <summary>
         /// Gets the kind of session error this exception represents.
         /// </summary>
+        /// <value>An enum value, describing the root cause of the error.</value>
```

Comment:
The "root cause" could be that the sysadmin who was supposed to keep the server running was lazy. I think this merely describes the error :-)